app/views/favorites/index.html.erb

```diff
@@ -1,12 +1,15 @@
 <header><h2 class='page-title'>Favorite Pets</h2></header>
 <section class='cards-container'>
-  <% unless @favorite_pets %>
+  <% if !@favorite_pets %>
```

Comment:
I had quite a bit of trouble with the logic here, this looks messy but it works, maybe could be refactored